server/api/audio_processor.py

```
@@ -0,0 +1,578 @@
+    def do_POST(self):
+        try:
+            # Safely validate and read request body
+            content_length = self._validate_content_length()
+            if content_length is None:
+                return  # Error response already sent
+                
+            body = self.rfile.read(content_length)
+            data = json.loads(body.decode('utf-8'))
+            
+            clip_ids = data.get('clip_ids', [])
+            background = data.get('background', False)
```

Ruff (0.12.2)

Local variable `background` is assigned to but never used

Remove assignment to unused variable `background`

(F841)